<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 226, 24);</string>
  </property>
  <widget class="QLabel" name="lbGameName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>JACKPOT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIndicacion">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seleccione las lineas que desea jugar</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb1A">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 120939.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 120939.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb1B">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121041.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121041.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb1C">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121057.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121057.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb2A">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121223.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121223.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb3A">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121240.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121240.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb2B">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121256.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121256.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb4A">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121313.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121313.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb3B">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>410</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121329.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121329.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb4B">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Fotos/Captura de pantalla 2020-10-04 121355.png</normaloff>../Fotos/Captura de pantalla 2020-10-04 121355.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbCantCreditos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>999999 Pesos</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCreditosApostar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Seleccione la cantidad de creditos a apostar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>542</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Â¡JUGAR!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
